docs/integrations.md

```diff
@@ -92,7 +92,7 @@ jobs:
         default: main
       semgrep_deployment_id:
         type: integer
-        default: 203
+        default: << DEPLOYMENT_ID >>
```

Comment:
Maybe instead of `<< DEPLOYMENT_ID >>`, which looks like it should syntactically parse, something that's an obvious placeholder, like `xxx` or `my deployment id`.